card/lib/card/format.rb

```diff
@@ -114,15 +114,15 @@ def include_set_format_modules
       end
     end
   
-    def inclusion_defaults
+    def inclusion_defaults nested_card
       @inclusion_defaults ||= begin
-        defaults = get_inclusion_defaults.clone
+        defaults = get_inclusion_defaults(nested_card).clone
         defaults.merge! @inclusion_opts if @inclusion_opts
         defaults
       end
```

Comment:
this is a little bit of a problem, conceptually at least (perhaps not in practice yet)

the @inclusion_defaults here is intended to be a little cache of this value *for the format object*.  but the inclusion defaults can differ by nested card, theoretically.

so if a given object has several nests, and the nested cards have different rules, this will break.

since that's not the case today on wikirate, we can deploy, but we should probably fix before 1.15